src/libopensc/pkcs15-esteid2025.c

```diff
@@ -0,0 +1,203 @@
+/*
+ * PKCS15 emulation layer for EstEID card issued from December 2025.
+ *
+ * Copyright (C) 2024, Raul Metsma <raul@metsma.ee>
```

Comment:
This is for the cards that will issued since the end of this year?

the copyright year should also be 2025 as we are couple of months into this year.

Reply:
Fixed